src/main/java/Duke.java

```diff
@@ -1,25 +1,54 @@
 import java.util.Scanner;
 
 public class Duke {
-    public static void main(String[] args) {
 
-        boolean isByeEntered = true;
-        String outputs;
+    public static final int MAX_INT = 100;
 
+    public static void greeting() {
         System.out.println(" Hello! I'm Duke");
         System.out.println(" What can I do for you?");
 
-        while(isByeEntered) {
+    }
+
+
+
+    public static void add() {
+        boolean isByeEntered = true;
+        String outputs;
+        String[] storeList;
+        storeList = new String[MAX_INT];
+        int i = 0;
+        int a = 1;
+
+        while (isByeEntered) {
             Scanner scan = new Scanner(System.in);
             outputs = scan.nextLine();
-            if(outputs.equals("bye")){
+            if (outputs.equals("bye")) {
                 System.out.println("Bye. Hope to see you again soon!");
                 return;
-            }else{
-            System.out.println(outputs);
+            } else if(outputs.equals("list")){
```

Comment:
Coding Standard Violation: Give space before and after statement
```suggestion
} else if(outputs.equals("list")) {

Reply:
same goes for line 30, 35